src/server/graphql/mutations/acceptTeamInviteNotification.js

```diff
@@ -8,22 +8,24 @@ export default {
   type: new GraphQLNonNull(NotifyAddedToTeam),
   description: 'Approve an outsider to join the organization',
   args: {
-    dbNotificationId: {
+    notificationId: {
```

Comment:
Hm, I don't know enough about the subject. UUIDs give you `2^128` possible values, while `short-id` gives you "7-14 url-friendly characters: A-Z, a-z, 0-9, _-" which is `(26 * 2 + 10 + 2)^14 == 2^84` possible values. I guess that means we'd need 2^84 global documents to see a conflict.

If we were to go the UUID route, we could use rethink's built in `r.uuid`. I hope (though I'm not sure) that UUIDs would be stored as integers rather than strings, which would perform better.

Reply:
rethinkdb uses strings by default, with dashes in em, it ain't great. I'm comfortable for using shortId for now. if we ever do hit a conflict, it should be as easy as swapping out shortid.generate() with something more robust.